rules/manager.go

```diff
@@ -625,3 +617,25 @@ func (m *Manager) AlertingRules() []*AlertingRule {
 	}
 	return alerts
 }
+
+// Implements prometheus.Collector.
```

Comment:
From the codacy check, should be `Describe Implements prometheus.Collector` Similarly below.